Sources/BrowserServicesKit/Autofill/AutofillUserScript+SecureVault.swift

```diff
@@ -19,6 +19,11 @@
 
 import WebKit
 
+public enum RequestVaultCredentialsAction: String, Codable {
```

Comment:
Given `.none` case always accompanies nil credentials, and `.fill` is always associated with credentials, how about changing the handler so that it only returns an action and parameterise the fill case with the credentials in question, then credentials don't need to be optional either.